scripts/run_notebooks.py

```diff
@@ -0,0 +1,55 @@
+"""Script to run all notebooks example notebooks.
+
+Taken from: https://github.com/pymc-labs/pymc-marketing/blob/main/scripts/run_notebooks/runner.py
+"""
+
+import logging
+from pathlib import Path
+
+import papermill
+from joblib import Parallel, delayed
+from tqdm import tqdm
+
+KERNEL_NAME: str = "python3"
+DOCS = Path("docs")
+NOTEBOOKS: list[Path] = [
+    # DOCS / "compare-fixest-pyfixest.ipynb",  # needs R
+    DOCS / "difference-in-differences.ipynb",
+    DOCS / "marginaleffects.ipynb",
+    DOCS / "quickstart.ipynb",
+    DOCS / "replicating-the-effect.ipynb",
+    DOCS / "stargazer.ipynb",  # failing notebook
```

Comment:
@s3alfisc  in Cell[22] there is an error in the notebook (checked locally)

```python
NotImplementedError: <class 'pyfixest.estimation.feols_.Feols'>
```

See https://github.com/py-econometrics/pyfixest/actions/runs/10707033524/job/29686144381